.circleci/config.yml

```diff
@@ -227,9 +249,18 @@ jobs:
     - restore_cache:
         keys:
         - *js_deps_cache_key
+    - restore_cache:
+        keys:
+        - *lint_cache_key
     - run:
         name: ESLint
         command: yarn lint:ts
+    - run:
+        name: MDLint
+        command: yarn lint:md
+    - save_cache:
+        paths: *lint_cache_paths
+        key: *lint_cache_key
```

Comment:
Can we add this back in now that the config is fixed?

Reply:
I did. It's added on line 260 and running here in 3.36s: https://circleci.com/gh/patternfly/patternfly-react/25984?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link